content/kubermatic/main/architecture/supported-providers/vsphere/vsphere.en.md

```diff
@@ -11,14 +11,13 @@ The Kubernetes vSphere driver contains bugs related to detaching volumes from of
 
```

Comment:
More a general note, something out of scope of this PR: 
All the provider pages are inconsistent, both in structure and content. Are there plans to improve that? 

E.g. what I miss here would be an introduction sentence e.g. 
```suggestion
KKP fully supports vSphere as a tier 1 provider. It allows the automatic provision of user-cluster nodes, and to dynamically manage storage.
```